addons/mail/static/src/components/discuss/discuss.xml

```diff
@@ -12,7 +12,7 @@
                 </t>
                 <t t-if="messaging.device.isSmall" t-call="mail.Discuss.content"/>
                 <t t-else="">
-                    <div t-attf-class="o_Discuss_content d-flex flex-column flex-grow-1 overflow-auto">
+                    <div t-attf-class="o_Discuss_content d-flex flex-column flex-grow-1 overflow-auto {{ o_Discuss_noThread_bg === True ? 'bg-white' : '' }}">
```

Comment:
`o_Discuss_noThread_bg` is useless: it's used in parent of `t-call` that actually sets it.
So here it will not put `bg-white`.

Reply:
Indeed. I've dropped the `t-set` used in the parent and I'm using the same condition seen in the parent. 👍